CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 2.19.2
+### Fixed
+- Task `Security Enlightn` will now only tell what composer packages have security issues, but will not block
```

Comment:
This was not broken AFAIK, so to me this feels like a (breaking) change, of which I'm not sure if we want to apply this. To my understanding people can also opt out of this in their local/project configuration
If we want this to happen I think this should be in 3.0 as until now we trust our systems to stop working once a a security issue is found

Reply:
Alright than i will close this pr and let people resolve it within their projects.